### ISSUE TITLE
index_stats is not working in 2.3.5

### ISSUE THREAD

**Issue by MatthewRalston** [helios/bioruby-samtools#60]
I've indexed a bam file with samtools and receive `true` on the indexed command and index_stats returns an empty hash.

**Comment by homonecloco** [comment]
Hi Mat, 
Can you provide an example? I am currently updating the gem to samtools-1.3.1 and it is passing the unit test for index_stat

**Comment by MatthewRalston** [comment]
So are you saying a version >2.3.5 is passing?

**Comment by homonecloco** [comment]
yes, but if you have any issues let us know. And if it fails, please attach an example so I can test it.

**Comment by MatthewRalston** [comment]
An example as in a bam file? When I initially had this issue, the choice of BAM file was not important; ie the index stats command produced an empty hash `{}` in several instances.

**Comment by homonecloco** [comment]
Can you check if it is working now? I couldn't reproduce the bug on the unit tests, and added a few more.
